src/components/CardList.js

```diff
@@ -0,0 +1,65 @@
+import baseImg from "../assets/base.png";
+
+function CardListItem(item) {
+    const createdAtDate = new Date(item.item.createdAt).toISOString().split("T")[0];
+
+    function timeAgo(timestamp) {
+        const currentDate = new Date();
+        const inputDate = new Date(timestamp);
```

Comment:
이런식으로 표현해볼수도 있을 것 같아요~
```suggestion
        const now = new Date();
        const target = new Date(timestamp);
```